src/main/java/org/folio/service/file/definition/FileDefinitionServiceImpl.java

```diff
@@ -32,4 +41,18 @@
   public Future<FileDefinition> update(FileDefinition fileDefinition, String tenantId) {
     return fileDefinitionDao.update(fileDefinition, tenantId);
   }
+
+  @Override
+  public Future<FileDefinition> findFileDefinition(String fileDefinitionId, String tenantId) {
```

Comment:
Can we just change the getById() to have this logic instead of a redundant method again?

Reply:
Yes, i moved the logic from findFileDefinition() method to the getByid()